file.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class User 
```

Comment:
Set a more purposeful name
```suggestion
class SuperUser
```